resources/views/account/accept/create.blade.php

```diff
@@ -94,6 +103,24 @@
 @section('moar_scripts')
 
     <script nonce="{{ csrf_token() }}">
+        $(document).ready(function(){
+            // Initially hide the div
+            $('#declined_msg').hide();
+            $('#declined_msg_label').hide();
+
```

Comment:
Nitpick/UX-Improvement: When the page is loaded there is a flash of the label and textarea popping up before disappearing. If the default state is for it to be hidden, can we add that class to the html so it doesn't appear before being hidden on page load?

Reply:
Im not able to get this to function properly when I do `class=hidden` on the divs for the inputs.

Reply:
Hm...I just tried and noticed the same thing. We must have some scoping somewhere that is breaking `.show()`? Either way...outside of the scope of this PR.